docs/products/flink/howto/real_time_alerting_solution.rst

```diff
@@ -152,7 +152,7 @@ This setup uses :doc:`windows </docs/products/flink/concepts/windows>` to determ
 
 1. Go to the **Data Tables** subtab.
 
-#. Select your Kafka service, enter ``CPU_OUT_AGG`` as the name, ``cpu_load_stats_agg`` as the topic, and the following as the SQL schema, then click **Create Table**:
+#. Select your Kafka service, use the default selection for connector type, key, and value data format, enter ``CPU_OUT_AGG`` as the name, ``cpu_load_stats_agg`` as the topic, and the following as the SQL schema, then click **Create Table**:
```

Comment:
"use the default selection for connector type" <- I'm not sure which is the default selection, could be mention the "standard Kafka connector type" and JSON for key and value data format?

Reply:
Yep, although I separate these cases into substeps, as otherwise there would be too many instructions included in a single step ;)